spec/features/articles_spec.rb

```diff
@@ -15,8 +15,14 @@
   end
   
   context "creating an article" do
+    let!(:user) { create :user, email: 'article@test.com' }
     before do
       visit '/articles'
+      click_link 'Log in'
+      fill_in 'Email', with: 'article@test.com'
+      fill_in 'Password', with: 'password'
+      click_button 'Log in'
```

Comment:
As your specs get more built out, this login code will be something you want to re-use. It's fine as it is now, but keep an eye on it and look for ways in the future to refactor this into a reusable method.